addon/models/twitch-video.js

```diff
@@ -0,0 +1,9 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+// import { belongsTo, hasMany } from 'ember-data/relationships';
+
+export default Model.extend({
+  twitchUser: attr(),
+  twitchTeam: attr(),
+  twitchSubscription: attr()
```

Comment:
I would generally prefer to be prescriptive in defining types for models, unless they are all objects that don't yet have models defined for them (and as such, can't have relationships). Even though the `attr` function doesn't require a type anymore, for an open-source library especially it's nice to be explicit for documentation's sake.